### ISSUE TITLE
Can't remove DRM from Adobe Digital Editions 4.5 Books

### ISSUE THREAD

**Issue by Murari1008** [apprenticeharper/DeDRM_tools#1142]
Does anyone know, how to remove DRM/Copy/Print from ADE 4.5 which contains EBX_Handler security V4?
Plz help......

**Comment by ElleKayEm** [comment]
If it can't be opened with ADE 2.0.1 with the same Adobe ID, then DeDRM can't remove the DRM yet. No idea if anyone is working on this. Not many vendors have switched to exclusively use the hardened DRM. Until it becomes more common, there's not a lot of motivation or even opportunity to look at it.  The way DeDRM works is to unlock the book with your credentials, not to “crack” the DRM.

**Comment by jsl303** [comment]
Any progress on this? MacOS Catalina can't run ADE 2.0.1.

**Comment by ElleKayEm** [comment]
No change.  If the ebookstore isn't using hardened Adobe DRM, you'll be fine with ADE 4.5.  Unfortunately I don't think there's any way to tell ahead of time.  Even if you were using ADE 2.0.1, you'd only know if you couldn't download it.

**Comment by jsl303** [comment]
I downloaded the book in 4.5, but DeDrm failed to decrypt. I assume it's 
using the new encryption...

Then there's no other option right?

**Comment by ElleKayEm** [comment]
You have ADE authorized with your Adobe ID and on the same machine and user as calibre with DeDRM?

**Comment by jsl303** [comment]
Yes. I can open the book in ADE 4.5, and I'm running Calibre with DeDRM 
on the same machine/user.

**Comment by jsl303** [comment]
Yes, I'm sure I did authorize. Other wise I won't be able to open and 
read the entire book on ADE right?

**Comment by ElleKayEm** [comment]
You'd be able to open and read it, but it would be tied to that installation of ADE.  You can see if it's authorized with your Adobe ID by looking at the Help menu > Authorization Information.  Then it is tied to the Adobe ID and can be read in other apps/devices authorized with the same Adobe ID.  And the key for your Adobe ID is used by DeDRM to remove the DRM.
